static/workflows/mobile-map-click-behavior-buffer.json

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
@ianschmitz I adopted a pattern of including workflow content as static assets and linking to them in a way which causes the file to be downloaded. A user can then import them into saas workflow at their leisure. What are your thoughts on this pattern?

Reply:
How are they used in the samples?

Reply:
Check out the note on [this article](https://deploy-preview-16--geocortex-dev-center.netlify.app/docs/web/usecases-change-default-map-click-behavior#extend-the-workflow-to-create-a-graphic-buffering-the-click-geometry) for an example

Reply:
Ah yeah that's pretty good.